zebra-chain/src/block/serialize.rs

```diff
@@ -162,3 +163,36 @@ impl AsRef<[u8]> for SerializedBlock {
         self.bytes.as_ref()
     }
 }
+
+/// A serialized transaction.
```

Comment:
Optional: does this belong in `transaction::serialize` instead?

Reply:
Ah that's right, moved in f1581bbf09